backend/src/schema/types/type/Post.gql

```diff
@@ -89,4 +91,11 @@ type Mutation {
     language: String
     categoryIds: [ID]
   ): Post
+  AddPostEmotions(to: _PostInput!, data: _EMOTEDInput!): EMOTED
+  RemovePostEmotions(to: _PostInput!, data: _EMOTEDInput!): EMOTED
+}
+
+type Query {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-06T20:19:48Z" title="Tuesday, August 6th 2019, 10:19:48 pm +02:00">Aug 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,4 +91,10 @@ type Mutation {
     language: String
     categoryIds: [ID]
   ): Post
+  RemovePostEmotions(from: _UserInput!, to: _PostInput!, data: _EMOTEDInput!): Boolean!
+}
+
+type Query {
+  postsEmotionsCountByEmotion(postId: ID!, data: _EMOTEDInput!): Int!
```


~I don't like STRANGECase, but I'm not going to request changes because of that - but you and everyone else are welcome to change it in the next PR :wink:~
Found improvements and would like to request changes - so you have the option to implement it in this PR :gift: 

Don't use STRANGECase, pleASE

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-07T05:01:07Z" title="Wednesday, August 7th 2019, 7:01:07 am +02:00">Aug 7, 2019</time>_

---


what's STRANGECase? I found this online https://www.artrix.co.uk/whats-on/all-shows/the-strange-case-of-dr-jekyll-and-mr-hyde

the case used in this example is called camel case and is used quite a lot in javascript, do you prefer pascal case?
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841